frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -87,6 +91,12 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
       item.subscription.metadata.name
     }?showDelete=true`;
 
+  const createLinkWorkflow =
```

Comment:
Instead of having separate `createLink` and `createLinkWorkflow` variables, it would be better to have a single `createLink` variables that's set to the right value.

```ts
const createLink =
  _.get(obj, ['metadata', 'annotations', 'marketplace.openshift.io/remote-workflow']) ||
  `/operatorhub/subscribe?pkg=${
    obj.metadata.name
  }&catalog=${catalogSource}&catalogNamespace=${catalogSourceNamespace}&targetNamespace=${namespace}`;

Reply:
Updated this.